test/syscalls/linux/proc_net.cc

```diff
@@ -184,11 +185,31 @@ TEST(ProcNetSnmp, TcpEstab) {
   EXPECT_EQ(oldPassiveOpens, newPassiveOpens - 1);
   EXPECT_EQ(oldCurrEstab, newCurrEstab - 2);
 
+  // Send 1 byte from client to server.
   ASSERT_THAT(send(s_connect.get(), "a", 1, 0), SyscallSucceedsWithValue(1));
 
+  constexpr int kPollTimeoutMs = 20000;  // Wait up to 20 seconds for the data.
+
```

Comment:
This seems really high for a test. This is all over loopback 20ms should be sufficient?

Reply:
Let's leave it at 20s then and we can fix it later if needed.